Automattic/wp-calypso
# Performance Profiler: Add an example email report to the banner

Resolves https://github.com/Automattic/dotcom-forge/issues/9151

|Logged-in | Logged-out|
|-------|------|
|![CleanShot 2024-09-16 at 18 00 21@2x](https://github.com/user-attachments/assets/a0f6021b-e979-4e79-8995-6b795d4affa3)| ![CleanShot 2024-09-16 at 18 00 13@2x](https://github.com/user-attachments/assets/6ed66ccd-7428-41c9-b3e2-30fd842c060a)|

## Proposed Changes

* Adds an example email report to the newsletter banner

## Why are these changes being made?


* To show the user what to expect when subscribing to the report

## Testing Instructions



* Apply this patch
* Open the profiler tool and run a test
* Check the newsletter banner contains an image as per Figma
* For small screens, the text should wrap as suggested here p1726564500775119/1726496035.364039-slack-C07AV9N4Y1H
* Check in Mobile view, the banner image is not displayed

## Pre-merge Checklist
